openapi/config-values.yaml

```diff
@@ -8,9 +8,32 @@ properties:
       - INFO
       - DEBUG
       - TRACE
-    description: Module log level
+    description: Module log level.
     default: DEBUG
   v3support:
     type: boolean
     default: false
-    description: NFS version v3 support. After enabling this setting, rpc-statd will be installed on nodes. When this setting is disabled, it will NOT be removed from the nodes.
+    description: NFS version v3 support.
```

Comment:
```suggestion
    description: NFS version v3 support. After enabling this setting, rpc-statd will be installed on nodes. When this setting is disabled, it will NOT be removed from the nodes.
```